packages/ethereum-storage/src/ethereum-storage.ts

```diff
@@ -225,6 +225,49 @@ export default class EthereumStorage implements StorageTypes.IStorage {
     };
   }
 
+  /**
+   * Append content into the storage: add the content to ipfs and the hash on Ethereum
```

Comment:
since it's prefixed with `_`, I guess it's "unstable"? should maybe be specified in the comments...